docs/xanadu_theme/static/xanadu.css_t

```diff
@@ -48,7 +48,8 @@ body {
   min-height: calc(100% - 50px);
   width: 300px;
   overflow: hidden;
-  box-shadow: 0 0px 0px 0 rgba(0,0,0,.05),0 30px 40px 0px rgba(0,0,0,.2) !important;
+  /*box-shadow: 0 0px 0px 0 rgba(0,0,0,.05),0 30px 40px 0px rgba(0,0,0,.2) !important;*/
+  border-right: 1px #ddd solid;
```

Comment:
(These stylistic changes are just to bring the theme in-line with the latest SF and PL documentation theme)